### PR TITLE
Fix formatting, move summary to remarks

### PR DESCRIPTION
**Pull request by JRAlexander**
Fix formatting, move summary to remarks 

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.models.onnxconverter.convert?branch=pr-en-us-33&view=ml-dotnet)

### CODE REVIEW

File: dotnet/xml/Microsoft.ML.Models/OnnxConverter.xml [hunk 1]
```diff
@@ -54,57 +54,43 @@
...
-             3. Logistic Regression
-             
-             See <a href="https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.Tests/OnnxTests.cs" />
-             for an example on how to train a model and then convert that model to ONNX.
-             </summary>
-        <remarks>To be added.</remarks>
+        <summary>Converts an ML.NET model to ONNX format by inspecting the transform pipeline from the end, checking for components that know how to save themselves as ONNX.
+        </summary>
+        <remarks>
+          <para><a href="https://onnx.ai/">ONNX</a> is an intermediate representation format for machine learning models. It is used to make models portable such that you can train a model using a toolkit and run it in another tookit's runtime, for example, you can create a model using ML.NET or any ONNX compatible toolkit, convert it to ONNX and then the ONNX model can be converted into say, CoreML, TensorFlow or WinML model to run on the respective runtime.
```
Comment (mairaw) [inline dotnet/xml/Microsoft.ML.Models/OnnxConverter.xml]: typo: tookit

nit: missing comma before last or

File: dotnet/xml/Microsoft.ML.Models/OnnxConverter.xml [hunk 2]
```diff
@@ -54,57 +54,43 @@
...
+          </para>
+          <para>This API supports the following arguments:
+             <list type="bullet">
+               <item><description><see cref="P:Microsoft.ML.Models.OnnxConverter.Onnx" /> indicates the file to write the ONNX protocol buffer file to. This is optional.</description></item>
+               <item><description><see cref="P:Microsoft.ML.Models.OnnxConverter.Json" /> indicates the file to write the JSON representation of the ONNX model. This is optional.</description></item>
+               <item><description><see cref="P:Microsoft.ML.Models.OnnxConverter.Name" /> indicates the name property in the ONNX model. If left unspecified, it will be the extension-less name of the file specified in the onnx indicates the protocol buffer file to write the ONNX representation to.</description></item>
```
Comment (mairaw) [inline dotnet/xml/Microsoft.ML.Models/OnnxConverter.xml]: shouldn't onnx be always capitalized?

Comment (JRAlexander) [reply dotnet/xml/Microsoft.ML.Models/OnnxConverter.xml]: Probably

File: dotnet/xml/Microsoft.ML.Models/OnnxConverter.xml [hunk 3]
```diff
@@ -54,57 +54,43 @@
...
+            </list>
+          </para>
+          <para>Learners that can be exported to ONNX:
+            <list type="number">
+              <item><description>FastTree</description></item>
+              <item><description>LightGBM</description></item>
+              <item><description>Logistic Regression</description></item>
+            </list>
+          </para>
+          <para>See <a href="https://github.com/dotnet/machinelearning/blob/master/test/Microsoft.ML.Tests/OnnxTests.cs">here</a> for an example on how to train a model and then convert that model to ONNX.
```
[the remark this answers is not in the document]
Comment (JRAlexander) [reply dotnet/xml/Microsoft.ML.Models/OnnxConverter.xml]: So should it be relative?

[the remark this answers is not in the document]
Comment (mairaw) [reply dotnet/xml/Microsoft.ML.Models/OnnxConverter.xml]: I don't think so since it's a GitHub link.

**Comment by JRAlexander** [comment]
@mairaw - all changes addressed. Thank you for the time spent reviewing and the thoughtful comments!

**Pull request merged**